README.md

```diff
@@ -3,6 +3,21 @@
 Rumba is [MDN's](https://developer.mozilla.org) new back-end. It supersedes [kuma](https://github.com/mdn/kuma) and
 mainly powers [MDN Plus](https://developer.mozilla.org/en-US/plus).
 
+## Quickstart
+
+Before you can start working with Rumba, you need to:
+
+1. Install [git](https://git-scm.com/) and [Rust](https://www.rust-lang.org/).
+2. Install additional dependencies:
+   - Mac OS (Intel): `brew install postgres libpq`
+   - Mac OS (M1): `arch -arm64 brew install postgres libpq`
+   - Ubuntu: `apt install gcc libpq-dev libssl-dev pkg-config`
+3. Run a PostgreSQL instance:
+   - Mac OS: e.g. [Postgres.app](https://postgresapp.com/)
+   - Docker: `docker run --name postgres -p 5432:5432 -e POSTGRES_USER=rumba -e POSTGRES_PASSWORD=rumba -e POSTGRES_DB=mdn -d postgres`).
+4. Copy `.settings.dev.toml` to `.settings.toml`.
+5. Run `cargo run`.
+
```

Comment:
Perhaps something like 
6. To create an authenticated session navigate to http://localhost:8000/users/fxa/login/authenticate/?next=%2F
7. To check you are logged in and ready to go navigate to http://localhost:8000/api/v1/whoami you should see your logged in user information.

Reply:
@Guyzeroth Wouldn't those steps only work with our client_id/client_secret?

PS: I tried it locally with http://localhost:8000/users/fxa/login/authenticate/?next=%2Fapi%2Fv1%2Fwhoami, but here's what I get after logging in: 
<img width="465" alt="image" src="https://user-images.githubusercontent.com/495429/185185315-328f6281-9c7a-466f-a08a-c08be867cbc2.png">

Reply:
Of course you are right. The service is rather useless without those creds however so perhaps we include a way to get hold of them? Your whoami endpoint will look like that if you aren't authenticated.

Reply:
What would be probably helpful is a way to simulate a FxA authentication via CLI. This would allow for community contributions, especially bug fixes.

Reply:
Resolved via 61623e98986d7014e309d4f5e22a3c4dbc6fefcb.